src/tools/tidy/src/fluent_used.rs

```diff
@@ -0,0 +1,59 @@
+//! Checks that all Flunt files have messages in alphabetical order
+
+use crate::walk::{filter_dirs, walk};
+use std::collections::HashMap;
+use std::path::Path;
+
+use regex::Regex;
+
+lazy_static::lazy_static! {
+    static ref MESSAGE: Regex = Regex::new(r#"(?m)^([a-zA-Z0-9_]+)\s*=\s*"#).unwrap();
+    static ref WORD: Regex = Regex::new(r"\w+").unwrap();
+}
+
+fn filter_fluent(path: &Path) -> bool {
```

Comment:
can this be moved into `fluent_alphabetical` to share some of the fluent handling code (and then to two separate things with the list of messages)?